service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/IgnoredResourcesUtils.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.adaptors.gp2gp.ehr.utils;
+
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.ResourceType;
+
+public class IgnoredResourcesUtils {
+    private static final List<ResourceType> IGNORED_RESOURCES = List.of(
```

Comment:
It doesn't matter much in this case BUT "in theory" a `Set` is more suitable for such case.